README.rst

```diff
@@ -20,15 +20,28 @@ Ccdproc can currently be installed via pip or from the source code.  For
 installation instructions, see the `online documentation`_ or docs/install.rst
 in this source distribution.
 
-This package is a work in progress...feel free to help out by picking an issue
-and working on it!
 
 Documentation is at `ccdproc.readthedocs.org
 <http://ccdproc.readthedocs.org/en/latest/>`_
 
+Contributing
+------------
+
+We have had the first stable release, but there is still plenty to do!  
+
+Please open a new issue or new pull request for bugs, feedback, or new features you would like to see.   If there is an issue you would like to work on, please leave a comment and we will be happy to assist.   New contributions and contributors are very welcome!
+
+New to github or open source projects?  If you are unsure about where to start or haven't used github before, please feel free to email `@crawfordsm`_ or `@mwcraig`_ .  We will more than happily help you make your first contribution. 
```

Comment:
From my experience, new people can be overwhelmed by the astropy dev docs.   But it might not be a bad idea to mention it and add a link to it, but we should emphasize emailing us as the first point of contact.